src/App.js

```diff
@@ -1,16 +1,74 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './components/TodoComponents/Todo.css'
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: '',
+    };
+  }
+
+  formHandler = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  }
+
+  addTodo = e => {
+    e.preventDefault();
+    if (this.state.task === '') return;
```

Comment:
well-done accounting for this edge case, but look into 2 ways you could have some info passed back to the user:
-you could use the `alert` method to send a custom message to the browser warning the user to fill out the field
-you could use the `required` attribute on the form's input element, which does the same thing but with a default method